Add --delete parameter

yuvipanda opened the pull request.

yuvipanda committed:
Add --delete parameter

ryanlovett reviewed (approved):
Looks straightforward. Maybe add a space so that it would be "Uploaded! -> Deleted!" and not "Uploaded!-> Deleted!". Also, without a carriage return, will "Deleted!" run onto the next print?

yuvipanda committed:
Leave a notice file in user home directories

- Users know how to ask for their files!
- Delete & leaving a notice file are now part of 'validate', so
  we aren't deleting files that haven't been uploaded! If something
  changes after the user home dir has been archived and uploaded,
  validation will fail and we can restart it. Much better than
  losing user data!
